.konflux/.gitignore

```diff
@@ -0,0 +1,2 @@
+# rocksdb source for main container builds in Konflux
```

Comment:
I am interested in opinions on this `$(repo_root)/.konflux` directory.
I did not want to put `rocksdb` sources anywhere deep.

I suggest that we can use some common root (like `.konflux`) for all downloaded files and Konflux-specific scripts. I.e. move `scripts/konflux` here.

That said, I'm not very happy with the directory name and not very sure about the idea hence asking for opinions.

Reply:
I created a README.md to help counter the last point. My suggestion would be to try land it in the current PR and then I'll follow up with one which moves `scripts/konflux` and downloaded blobs to this dir. If the end result will look unsatisfactory, I'll re-organize it for `main`. Sounds good?

Reply:
Yes, that sounds good to me.